src/js/components/Button/StyledButton.js

```diff
@@ -5,6 +5,7 @@ import {
   backgroundStyle,
   disabledStyle,
   focusStyle,
+  unfocusStyle,
```

Comment:
I think this functionality looks great and works as expected. Wondering if we should update the comment above `unfocusStyle` to explain why this is a valid use case of the function: https://github.com/grommet/grommet/blob/master/src/js/utils/styles.js#L331

May be helpful add the link to the blog article you referenced in the comment, it was a helpful read: https://nelo.is/writing/styling-better-focus-states/

Reply:
good idea! I will update the comment accordingly

Reply:
i forgot to update the PR. ive just now changed the unfocusStyle to include the new description. sorry about the delay